<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="lapapotte" name="com.google.refine.extension.database.cmd.ExecuteQueryCommandTest" tests="1" failures="0" timestamp="22 Nov 2018 18:06:26 GMT" time="0.002" errors="1">
  <testcase name="testDoPost" time="0.002" classname="com.google.refine.extension.database.cmd.ExecuteQueryCommandTest">
    <error type="com.google.refine.extension.database.DatabaseServiceException" message="Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connexion refusée (Connection refused)">
      <![CDATA[com.google.refine.extension.database.DatabaseServiceException: Could not connect to address=(host=127.0.0.1)(port=3306)(type=master) : Connexion refusée (Connection refused)
	at com.google.refine.extension.database.mysql.MySQLConnectionManager.getConnection(MySQLConnectionManager.java:157)
	at com.google.refine.extension.database.mysql.MySQLDatabaseService.getConnection(MySQLDatabaseService.java:302)
	at com.google.refine.extension.database.DBExtensionTestUtils.initTestData(DBExtensionTestUtils.java:120)
	at com.google.refine.extension.database.DatabaseTestConfig.beforeSuite(DatabaseTestConfig.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testDoPost -->
</testsuite> <!-- com.google.refine.extension.database.cmd.ExecuteQueryCommandTest -->
